manifests/install.pp

```diff
@@ -1,110 +1,131 @@
 class ss_graphite_client::install inherits ss_graphite_client {
 
   package {
-    realpath: ensure => installed;
-    logtail: ensure => installed;
-    logster: ensure => installed;
+    'logtail': ensure => installed;
   }
-
-  file {"/usr/lib/python2.7/dist-packages/logster/parsers":
+  
+  #package names are unstable between releases
+  if $facts['lsbdistcodename'] == 'jessie' {
+    package {
+    'realpath': ensure => installed;
+    'logster': ensure => installed;
+    }
+  }
+  #logster no longer included as of stretch
+  #so we can install via pip and github link
+  if $facts['lsbdistcodename'] == 'buster' {
+    file { 'git_proxy':
+      path => '/root/.gitconfig',
+      content => "[http] \n proxy = ${::params::http_proxy}",
+      before => Exec['installLogster']
+    }
+    $logster_vcs = 'https://github.com/etsy/logster.git'
+    $logster_ref = 'a24249886391a8b885d21c882f6fcaa95e29b015'
+    exec { 'installLogster':
+      command => "pip install --proxy ${::params::http_proxy} git+${logster_vcs}@${logster_ref}#egg=logster",
```

Comment:
Take a look at stdlib

https://ask.puppet.com/question/25819/how-to-install-pip-packages/

Should be able to manage proxy better too

Reply:
Just tested - the logster package from stretch and jessie installs fine on buster too.

Minimal dependencies:
`Depends: logtail, python-pkg-resources, python:any (>= 2.7.5-5~), python:any (<< 2.8), python`

Reply:
i think pulling in the logster deb package makes sense to me

Reply:
cloning requires github to be up so that's not ideal to have puppet runs fail just because github is down